.github/workflows/goth-nightly.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Configure node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16
```

Comment:
Why are we changing version?

Reply:
node16 is no longer supported. We are planning to change versions to 18 everywhere and when I touched the pipeline file I thought I would change it.